geomesa-core/src/main/scala/org/locationtech/geomesa/core/data/AccumuloDataStore.scala

```diff
@@ -355,8 +344,10 @@ class AccumuloDataStore(val connector: Connector,
                                       featureType: SimpleFeatureType,
                                       tableName: String) {
 
-    val splits = (1 to maxShard).map { i => s"%0${maxShard.toString.length}d".format(i) }.map(new Text(_))
-    tableOps.addSplits(tableName, new java.util.TreeSet(splits))
+    if (maxShard > 1) {
+      val splits = (1 to maxShard - 1).map { i => s"%0${maxShard.toString.length}d".format(i) }.map(new Text(_))
```

Comment:
(Picky) Go ahead and create the 'new Text' inside th first map instead of using two?